src/ansys/aedt/core/application/analysis.py

```diff
@@ -1396,18 +1396,34 @@ def _create_setup(self, name="MySetupAuto", setup_type=None, props=None):
 
             if self.solution_type == "SBR+":
                 setup.auto_update = False
-                default_sbr_setup = {
-                    "RayDensityPerWavelength": 4,
-                    "MaxNumberOfBounces": 5,
-                    "EnableCWRays": False,
-                    "EnableSBRSelfCoupling": False,
-                    "UseSBRAdvOptionsGOBlockage": False,
-                    "UseSBRAdvOptionsWedges": False,
-                    "PTDUTDSimulationSettings": "None",
-                    "SkipSBRSolveDuringAdaptivePasses": True,
-                    "UseSBREnhancedRadiatedPowerCalculation": False,
-                    "AdaptFEBIWithRadiation": False,
-                }
+                if settings.aedt_version >= "2024.2":
+                    default_sbr_setup = {
+                        "RayDensityPerWavelength": 4,
+                        "MaxNumberOfBounces": 5,
+                        "IsMonostaticRCS:": True,
+                        "EnableCWRays": False,
+                        "FastFrequencyLooping": False,
+                        "EnableSBRSelfCoupling": False,
+                        "UseSBRAdvOptionsGOBlockage": False,
+                        "UseSBRAdvOptionsWedges": False,
+                        "PTDUTDSimulationSettings": "None",
+                        "SkipSBRSolveDuringAdaptivePasses": True,
+                        "UseSBREnhancedRadiatedPowerCalculation": False,
+                        "UseSBRAdvOptionsNFAccuracy": False,
+                    }
+                else:
+                    default_sbr_setup = {
+                        "RayDensityPerWavelength": 4,
+                        "MaxNumberOfBounces": 5,
+                        "EnableCWRays": False,
+                        "EnableSBRSelfCoupling": False,
+                        "UseSBRAdvOptionsGOBlockage": False,
+                        "UseSBRAdvOptionsWedges": False,
+                        "PTDUTDSimulationSettings": "None",
+                        "SkipSBRSolveDuringAdaptivePasses": True,
+                        "UseSBREnhancedRadiatedPowerCalculation": False,
+                        "AdaptFEBIWithRadiation": False,
+                    }
```

Comment:
```suggestion
                    default_sbr_setup = {
                        "RayDensityPerWavelength": 4,
                        "MaxNumberOfBounces": 5,
                        "EnableCWRays": False,
                        "EnableSBRSelfCoupling": False,
                        "UseSBRAdvOptionsGOBlockage": False,
                        "UseSBRAdvOptionsWedges": False,
                        "PTDUTDSimulationSettings": "None",
                        "SkipSBRSolveDuringAdaptivePasses": True,
                        "UseSBREnhancedRadiatedPowerCalculation": False,
                        "AdaptFEBIWithRadiation": False,
                    }
                    if settings.aedt_version >= "2024.2":
                        default_sbr_setup["IsMonostaticRCS"] = True
                        default_sbr_setup["FastFrequencyLooping"] = False
```
This should help to distinguish the changes between each version